api/app/db/models/thessian_polygon_areas.py

```diff
@@ -0,0 +1,19 @@
+""" Class models that reflect resources and map to database tables for HFI Calculator.
+"""
+from geoalchemy2.types import Geometry
+from sqlalchemy import (Column, Integer,
+                        Sequence)
+from app.db.database import Base
+
+
+class ThessianPolygonArea(Base):
+    """ Thessian polygon for a given area """
+    __tablename__ = 'fire_area_thessian_polygons'
+
+    id = Column(Integer, Sequence('thessian_area_id_seq'),
+                primary_key=True, nullable=False, index=True)
+    polygon = Column(Geometry(geometry_type='POLYGON', from_text='ST_GeomFromGeoJSON'))
```

Comment:
You need to be explicit about the creation of an index on the geometry.  You'll need something like
`
Index('idx_fire_area_thessian_polygons_geom',
      ThessianPolygonArea.geom, postgresql_using='gist')`

in the global scope. See api/app/db/models/weather_models.py

If your not explicit, then subsequent runs of alembic autogenerate will create a migration to remove the index (unless that's been fixed since!)

Reply:
```suggestion
    geom = Column(Geometry(geometry_type='POLYGON', from_text='ST_GeomFromGeoJSON'))
```
Convention is to call the geometry column "geom"

Reply:
I think having the from_text='ST_GeomFromGeoJSON' implies that the data is stored as json. We don't want to store it as json because then we lose a lot of the veratility that postgis gives us, we want this to be binary. I'm not sure about this though - can you check?

Reply:
Cannot seem to find docs on how postgis stores geometries.

https://postgis.net/docs/ST_GeomFromGeoJSON.html
Above says "Constructs a PostGIS geometry object from the GeoJSON representation." It reads to me as if PostGIS stores it one way but specifying a different ST geom data type function will marshall inputs/outputs using the function specified.

By default alembic seems to use `from_text='ST_GeomFromEWKT'`

Reply:
Ok - that seems to make sense. I may just tinker with that when I do the shapefile thing - that's actually pretty cool if that's how it works - align with your input and output.